### ISSUE TITLE
rpm repository doesn't works properly!

### ISSUE THREAD

**Issue by bitchecker** [qTox/qTox#1326]
as i wrote here ( https://github.com/tux3/qTox/issues/1324#issuecomment-76700573 )
into official rpm repo there are so much qtox versions but system install 382 also if into repository there are newest version (459)!

i've reported so much issues ignoring that repository doesn't works as it must to do, so it can be possible that so much reported issues should be closed after testing the real latest version!

reported on Fedora 21 with XFCE

**Comment by zetok** [comment]
probably fixed, could you check and close this issue if that's the case?

**Comment by bitchecker** [comment]
no updates founds!

i tried to remove and reinstall but always 382 version is installed and not current 459!!

**Comment by stqism** [comment]
We were able to figure out his issue was because there were actually no updates because he had the latest and mistook 1382 for a version in a name.

Tl;Dr: User error that fixed itself on his end when I made it build a new rpm.
